### DISCUSSION TITLE
Import txto in MTex & EBSD

### DISCUSSION THREAD

**Discussion by MehrzadSoltani** [mtex-toolbox/mtex#773] [Ask Anything]
Hello,

1. I follow the instruction in the link (https://icme.hpc.msstate.edu/mediawiki/index.php/MTEX_Tutorial.html). I import texture.txto, but it doesn't work. I am faced with error (Could not detect file format...)

![screenshot 3](https://user-images.githubusercontent.com/79067007/107916222-8f192a00-6f2b-11eb-904e-ff8e5c8de627.jpg)

2. Another question is that which file extended by Abaqus has to be used as EBSD in MTex?

Best,

**Comment by kilir** [comment]
Hi,

wrt 1., maybe you have to share the input you use.

Cheers,
Rüdiger

**Comment by MehrzadSoltani** [comment]
Hi,

I don't know how to share txto file here. Because this file type isn't supported to share here.

Best,

**Reply by kilir** [reply]
How about packing it as a zip?

Rüdiger

**Comment by MehrzadSoltani** [comment]
Thanks. I hope it works.
[texture.zip](https://github.com/mtex-toolbox/mtex/files/5983288/texture.zip)

**Reply by kilir** [reply]
Hi,
well, your file does not really like an odf file to me, rather a list of 500 orientations which are repeated 8 times which again is repeated 7 times. Later on again, Euler angles are listed in blocks of 500 assigned "...at increment" are slightly changed, blocks are repeated a couple of times. So if you know what those should mean, it's straight forward to load and and present those in mtex.

```
fname = 'texture.txto'
CS = crystalSymmetry('432')
ori   = loadOrientation_generic(fname,'cs',CS, ...
        'ColumnNames',{'Psi' 'Theta' 'Phi'}, ...
        'Column', [1 2 3],'Kocks')
   
h = [Miller({1,0,0},{1,1,0},{1,1,1},CS)]
plotPDF(ori(1:500),h,'contourf')
```

Cheers,
Rüdiger
